package.json

```diff
@@ -43,6 +43,18 @@
     "autoprefixer": "^9.8.6",
     "jest-dom": "^4.0.0",
     "postcss": "^7.0.36",
-    "react-test-renderer": "^17.0.2"
+    "react-test-renderer": "^17.0.2",
+    "babel-eslint": "^10.1.0",
```

Comment:
@padmajabhol just some minor changes

We don't want all the eslint and prettier related dependencies in production. It's just to maintain a uniform style while coding locally. If I were you, I would have installed them as dev-dependencies.